pyvo/utils/xml/elements.py

```diff
@@ -125,7 +177,7 @@ def object_attrs(obj):
     objtype = type(obj)
     attrs = {
         getattr(objtype, name).name: value for name, value in getmembers(obj)
-        if isinstance(getattr(objtype, name, None), xmlattribute)}
+        if value and isinstance(getattr(objtype, name, None), xmlattribute)}
```

Comment:
What's the story with this and :194?  Seems like there was maybe some parsing trouble?